### PR TITLE
Added load test for Thunder

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Below test case is added in this pull request :
thunder_loadtest.test.js

### CODE REVIEW

File: src/tests/thunder/thunder_loadtest.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          }, 1000)
+        })
+      },
+    },
+    {
+      title: 'Repeat the steps for 1 hour',
+      description: 'Nested test to repeat the test for 1 hour',
+      repeat: {
+        seconds: 1 * 60 * 60, //One hour
+      },
```
Comment (wouterlucas) [inline src/tests/thunder/thunder_loadtest.test.js:65]: You need to keep reloading thunder UI during that one hour repeat. So if you move the first 2 steps within the repeat the test is great!

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/tests/thunder/thunder_loadtest.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/tests/thunder/thunder_loadtest.test.js:65]: @wouterlucas , 
Updated the code as per the above comment. Please review and let me know your feedback if any

Comment (wouterlucas) [reply src/tests/thunder/thunder_loadtest.test.js:65]: Great! Thanks, just one remark when we set the url again does the browser actually reload the page? or do we navigate away and back to force a full refresh of the UI?

Comment (sandeep-vedam) [reply src/tests/thunder/thunder_loadtest.test.js:65]: @wouterlucas , Browser loads full fresh ThunderUI(the URl we set).

Comment (wouterlucas) [reply src/tests/thunder/thunder_loadtest.test.js:65]: Thanks! Perfect.

**Review by wouterlucas — APPROVED** [review]

**Pull request merged**